packages/styles/gulp/build/default.js

```diff
@@ -8,10 +8,17 @@
 'use strict';
 
 const gulp = require('gulp');
+const sass = require('../tasks/sass');
 
 /**
  * Gulp task export
  *
  * @module default
  */
-module.exports = gulp.task('default', gulp.series('clean', 'sass'));
+module.exports = 
```

Comment:
What did you want to point `module.exports` to...?

Reply:
@IgnacioBecerra can you take a look? The module.exports looks off like what @asudoh mentioned.

Reply:
@asudoh @jeffchew Separate tasks into their own files and exported the default tasks correctly this time around. 👍